default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub , rustPlatform , pkg-config, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ord";
+  version = "0.5.0";
```

Comment:
Can we avoid hardcoding a version here? It would be nice if we didn't need to keep this version in sync with the `Cargo.toml`